### PR TITLE
What, why & screenshots in readme

### LINKED ISSUE #57: Update Desktop readme with "what & why" + screenshot(s)
Liven up the `README.md` a bit with:
- [x] Better explanation of what Hypergraph Desktop does
- [x] Better explanation of why Hypergraph Desktop exists
- [x] Screenshots of Hypergraph Desktop

### PR DESCRIPTION
**Pull request by jameslibscie**
To give at least *some* info regarding the product ;)

2nd paragraph comes from the welcome screens. I suggest we keep those texts fairly in sync for now as they serve a very similar purpose.

closes #57

### CODE REVIEW

File: docs/screenshots/.gitkeep [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (juliangruber) [inline docs/screenshots/.gitkeep:1]: why did you add this file if the folder isn't empty?

Comment (jameslibscie) [reply docs/screenshots/.gitkeep:1]: Afaik, the GitHub web interface doesn't allow you to upload files to new folders - only to create files in new folders. I forgot to delete this - good catch :)
